ERCS/erc-7704.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 7704
+title: Soul Resonance Token
+description: this EIP defines an interface for Soul Resonance Tokens
+author: Mike Smith (@mike0x1024), Abe Johnson <abe@burve.io>, Alvis Kaiser <alvis@burve.io>, WurDst Tsao <wurst@burve.io>
+discussions-to: https://ethereum-magicians.org/t/eip-7704-soul-resonance-token/20058
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-28
+requires: 20
+---
+
+## Abstract
+
+This EIP proposes the following improvements to Soul Bond Tokens:
+Extensibility to [ERC-20](./eip-20.md).
+Token Minting and Burning while preserving non-transferability.
```

Comment:
Your abstract should be complete sentences. Perhaps:

```suggestion
This EIP proposes the following improvements to Soul Bound Tokens: extensibility to [ERC-20](./eip-20.md), and token minting and burning while preserving non-transferability.
```

I would also recommend going into the technical side a bit more. Give the reader a high level understanding of how your proposal accomplishes its goals.